chain/cosmos/gas.go

```diff
@@ -0,0 +1,38 @@
+package cosmos
+
+import (
+	"context"
+
+	"github.com/renproject/pack"
+)
+
+// A GasEstimator returns the gas-per-byte that is needed in order to confirm
+// transactions with relatively high speed. In distributed networks that work to
+// collectively build transactions, it is important that all nodes return the
+// same values from this interface.
+type GasEstimator interface {
+	GasPerByte(ctx context.Context) (pack.U64, error)
+	GasPerSignature(ctx context.Context) (pack.U64, error)
```

Comment:
This method does not seem to actually be called anywhere. What is it for, and can we remove it?